wdl/ExpansionHunter.wdl

```diff
@@ -86,6 +86,17 @@ task RunExpansionHunter {
     command <<<
         set -euxo pipefail
 
+        BAM_OR_CRAM_DIR="$(dirname "~{bam_or_cram}")"
+        BAM_OR_CRAM_INDEX_FILENAME="$(basename "~{bam_or_cram_index}")"
+        DEST="$BAM_OR_CRAM_DIR/$BAM_OR_CRAM_INDEX_FILENAME"
+        if [ $DEST != ~{bam_or_cram_index} ]; then
+            mv ~{bam_or_cram_index} $DEST
+        fi
```

Comment:
Do you need the `if`?

Reply:
in my tests, `mv` exits with fail code if the file is already there.